docker-compose.yml

```
@@ -1,6 +1,6 @@
 services:
   qgis-server-light:
-    image: opengisch/qgis-server-light:dev
+    image: ghcr.io/opengisch/qgis-server-light-dev:latest
```

```suggestion
    image: opengisch/qgis-server-light-dev:latest
```

Following the discussion on https://github.com/opengisch/georama/issues/106